helpers/theme.js

```diff
@@ -7,12 +7,14 @@ const {
 	ALERT_STATUS_URL,
 } = require('../config/global');
 
-const instanceSpecificTitle = () => {
+const instanceSpecificTitle = (short = false) => {
 	switch (SC_THEME) {
 		case 'n21':
 			return 'Niedersächsischen Bildungscloud';
+		case 'brb':
+			return 'Schul-Cloud Brandenburg';
 		default:
-			return SC_TITLE;
+			return short ? SC_SHORT_TITLE : SC_TITLE;
```

Comment:
still dont see the reason for this adjustment for BRB. 
the instance specific title for BRB is stored directly in the variable `SC_TITLE` and does not need to be adjusted due to conjugation and so the usage of `shortThemeTitle` below becomes unnecessary. If I missunderstood something totally lets talk about it directly.